{{cookiecutter.github_repository}}/settings/production.py

```diff
@@ -41,9 +41,21 @@
 MANAGERS = ADMINS
 {%- if cookiecutter.add_django_cors_headers.lower() == 'y' %}
 
-# cors
+# CORS
 # --------------------------------------------------------------------------
 CORS_ORIGIN_WHITELIST = env.list('CORS_ORIGIN_WHITELIST')
+CORS_ALLOW_HEADERS = (
+    'accept',
+    'accept-encoding',
+    'authorization',
+    'content-type',
+    'dnt',
+    'origin',
+    'user-agent',
+    'x-csrftoken',
+    'x-requested-with',
+    'access-control-allow-origin',
+)
```

Comment:
I think this should go to common settings, and get applied to development as well.

Reply:
Also, I might import the defaults from the package itself and add the `access-control-allow-origin` to it, instead of copying over all the fields.

Submit upstream fix as well?

Reply:
Alright, makes sense. On it!